.github/workflows/merged.yml

```diff
@@ -84,50 +174,77 @@ jobs:
           # import certificates from secrets
           echo -n "${{env.AWS_SECRET_DISTRIBUTION_P12_ENCODED}}" | base64 --decode -o $CERTIFICATE_PATH
           echo -n "${{env.AWS_SECRET_AUTH_KEY_P8_ENCODED}}" | base64 --decode -o $APIKEY_PATH
-          
+
           bundle exec fastlane prerelease configuration:"Staging" \
             certificate_path:$CERTIFICATE_PATH \
             certificate_password:$CERTIFICATE_PASSWORD \
             apikey_path:$APIKEY_PATH --verbose
-            
+
           bundle exec fastlane prerelease configuration:"Build" \
             certificate_path:$CERTIFICATE_PATH \
             certificate_password:$CERTIFICATE_PASSWORD \
             apikey_path:$APIKEY_PATH --verbose
-          
+
+  manual_deploy_app:
+    if: github.event_name == 'workflow_dispatch'
+    needs: [test_${{ inputs.build_configuration }}_config, test_ui]
```

Comment:
Slightly unsure whether this would work, I found [this GitHub article](https://docs.github.com/en/enterprise-cloud@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputs) on lowercasing the inputs from the `build_configuration` workflow which would be perfect but I'm not 100% sure

Reply:
I imagine there's a notable mention here of lowercasing the parameters "during runtime" which wouldn't be appropriate to construct workflow names within the expected workflow array

Reply:
Sorry Dylan, didn't quite understand this comment, what do you think might not work? Deploying the apps to App Store Connect?

Reply:
Update: removed manual deploy action from workflow until this is needed